pkg/security/podsecuritypolicy/provider_test.go

```diff
@@ -639,25 +639,6 @@ func TestValidatePodSecurityContextSuccess(t *testing.T) {
 }
 
 func TestValidateContainerSecurityContextSuccess(t *testing.T) {
-	var notPriv bool = false
-	defaultPod := func() *api.Pod {
```

Comment:
We already have the same function in this file: https://github.com/kubernetes/kubernetes/blob/6a0d3c74949385494e42e5f9142925e851991ac8/pkg/security/podsecuritypolicy/provider_test.go#L928-L950